src/Event/ControllerListener.php

```diff
@@ -60,13 +60,6 @@ public function onKernelController(FilterControllerEvent $event): void
         $route = $request->attributes->get('_route');
         $controller = $event->getController();
         $service = $this->groupService;
-        $syliusConfig = $request->attributes->get('_sylius');
-        if (!empty($syliusConfig['permission'])) {
-            $syliusPermission = $syliusConfig['permission'];
-            if (!empty($syliusPermission && $syliusPermission === true)) {
-                return;
-            }
-        }
```

Comment:
In my opinion, the only change should be : 

`if (!empty($syliusPermission) || $syliusPermission === false)) {`

Because we don't want routes without `permission: true` to be validated by this.